src/_companies/europe/united-kingdom/london/easol/index.md

```diff
@@ -0,0 +1,38 @@
+---
+name: Easol
+logo: logo.png
+
+continent: Europe
+country: United Kingdom
+region: London
+city: London
+
+website: https://easol.com
+careers_page: https://easol.com/careers
+
+work_arrangements:
+- onsite
```

Comment:
YAML arrays should be indented one level deeper than the key ([source](https://www.w3schools.io/file/yaml-arrays/)). Surprisingly, it doesn't seem to be an issue here, but YAML can be quite picky with its indentation.
```suggestion
  - onsite
```

Reply:
Both are valid according to the specification but I'm happy to follow your direction.

Perhaps configuring and enabling a YAML linter for this would reduce the need for future human intervention?

Reply:
> Both are valid according to the specification but I'm happy to follow your direction.
> 
> Perhaps configuring and enabling a YAML linter for this would reduce the need for future human intervention?

That's a good idea, I'll look into that